content/working_with/console/widgets/deployments.md

```diff
@@ -65,7 +65,17 @@ The hamburger menu on the right of every deployment allows performing the follow
 3. Click **Execute**.
 
 You can also use **Install** or **Uninstall** menu options to execute those specific workflows.
+For these two workflows you will also be able to track the progress of the execution as at the bottom of the deployment row there will be thin line visible. Progress is counted basing on number of node instances installed (in case of install workflow) or deleted (in case of uninstall workflow). 
```

Comment:
```suggestion
For these two workflows you will also be able to track the progress of the execution as at the bottom of the deployment row there will be thin line visible. Progress is calculated based on number of node instances installed (in case of install workflow) or deleted (in case of uninstall workflow). 
```